tests/execution/test_executors.py

```diff
@@ -230,24 +229,20 @@ def test_get_execotor_for_tasks_default_execution_manager(purpose, check):
 
 
 def test_end_to_end_parallelizable_with_input_in_collect():
-    def par() -> Parallelizable[int]:
-        for i in range(10):
-            yield i
-
-    def identity(par: int) -> int:
-        return par
-
-    def foo() -> int:
-        return 1
+    dr = (
+        driver.Builder()
+        .with_modules(inputs_in_collect)
+        .enable_dynamic_execution(allow_experimental_mode=True)
+        .with_remote_executor(SynchronousLocalTaskExecutor())
+    ).build()
+    assert dr.execute(["collect"])["collect"] == 46
 
-    def collect(identity: Collect[int], foo: int) -> int:
-        return sum(identity) + foo
 
-    tmp_module = create_temporary_module(par, identity, collect, foo)
+def test_end_to_end_parallelizable_with_overrides():
     dr = (
         driver.Builder()
-        .with_modules(tmp_module)
+        .with_modules(inputs_in_collect)
         .enable_dynamic_execution(allow_experimental_mode=True)
         .with_remote_executor(SynchronousLocalTaskExecutor())
     ).build()
-    assert dr.execute(["collect"])["collect"] == 46
+    assert dr.execute(["collect"], overrides={"par": [1, 2, 3]})["collect"] == 7
```

Comment:
this doesn't test overriding collect though?

Reply:
can you override identity?

Reply:
Nope, you cannot override `identity` -- its in the middle of the block. Can test overriding `collect` as well.

Reply:
Yes, but I don't have the time right now to dig in on that -- this is a patch and why its still in experimental. Will create an issue.

Reply:
https://github.com/DAGWorks-Inc/hamilton/issues/301